packages/fetchai/skills/erc1155_client/tests/test_behaviours.py

```diff
@@ -52,7 +52,8 @@ def test_setup(self):
     def test_act_is_searching(self):
         """Test the act method of the search behaviour where is_searching is True."""
         # setup
-        self.strategy._is_searching = True
+        self.strategy.is_searching = True
+        assert self.strategy.is_searching
```

Comment:
the assert I should have removed. I used it to verify prior to setting it directly that this is indeed the issue, there is no private `_is_searching` and property `is_searching`